WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptModalView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+struct POSSendReceiptModalView: View {
+    @EnvironmentObject private var posModel: PointOfSaleAggregateModel
+    @Environment(\.dismiss) private var dismiss
```

Comment:
Is this going to stay in a sheet? If we move to a posModal this will obviously need to have a binding to an isPresented to control presentation.

Reply:
I'm not sure, designs are still TBD so I went ahead with presenting a view for simplicity, encapsulated to the TotalsView, if we move ahead with the design I think we can add it as `.posModal` and trigger the presentation from `posModel.shouldSomething`, which I'm guessing should also make it more testable.